### PR TITLE
Chore: Wishlist - Replace Block Escaping with Escaper

### LINKED ISSUE #37147: [Issue] Chore: Wishlist - Replace Block Escaping with Escaper
This issue is automatically created based on existing pull request: magento/magento2#37042: Chore: Wishlist - Replace Block Escaping with Escaper

---------
### Description (*)
Refactors the `Magento_Wishlist` module to replace `$block` escaping functions with `$escaper` escaping functions.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [x] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by pykettk**
### Description (*)
Refactors the `Magento_Wishlist` module to replace `$block` escaping functions with `$escaper` escaping functions.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [x] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#37147: Chore: Wishlist - Replace Block Escaping with Escaper

### CODE REVIEW

**Comment by pykettk** [comment]
@magento create issue

**Comment by pykettk** [comment]
@magento run all tests

File: app/code/Magento/Wishlist/view/frontend/templates/item/column/edit.phtml [hunk 1]
```diff
@@ -3,16 +3,20 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/item/column/edit.phtml:6]: missing new line

File: app/code/Magento/Wishlist/view/adminhtml/templates/customer/edit/tab/wishlist.phtml [hunk 2]
```diff
@@ -3,11 +3,17 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/adminhtml/templates/customer/edit/tab/wishlist.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/base/templates/product/price/bundle/configured_price.phtml [hunk 3]
```diff
@@ -3,9 +3,13 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
-?>
-<?php
-/** @var \Magento\Catalog\Pricing\Render\ConfiguredPriceBox $block */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/base/templates/product/price/bundle/configured_price.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/base/templates/product/price/configurable/configured_price.phtml [hunk 4]
```diff
@@ -3,12 +3,18 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
-?>
-<?php
-/** @var \Magento\Wishlist\Pricing\Render\ConfiguredPriceBox $block */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/base/templates/product/price/configurable/configured_price.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/button/share.phtml [hunk 5]
```diff
@@ -3,11 +3,19 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/button/share.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/button/tocart.phtml [hunk 6]
```diff
@@ -3,12 +3,20 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/button/tocart.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/button/update.phtml [hunk 7]
```diff
@@ -3,12 +3,19 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/button/update.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/cart/item/renderer/actions/move_to_wishlist.phtml [hunk 8]
```diff
@@ -3,13 +3,18 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/cart/item/renderer/actions/move_to_wishlist.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/catalog/product/view/addto/wishlist.phtml [hunk 9]
```diff
@@ -3,14 +3,19 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/catalog/product/view/addto/wishlist.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/email/items.phtml [hunk 10]
```diff
@@ -3,8 +3,13 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/email/items.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/item/column/actions.phtml [hunk 11]
```diff
@@ -3,12 +3,17 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/item/column/actions.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/item/column/cart.phtml [hunk 12]
```diff
@@ -3,13 +3,20 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/item/column/cart.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/item/column/image.phtml [hunk 13]
```diff
@@ -3,13 +3,18 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/item/column/image.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/item/column/name.phtml [hunk 14]
```diff
@@ -3,15 +3,20 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/item/column/name.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/item/column/remove.phtml [hunk 15]
```diff
@@ -3,10 +3,18 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/item/column/remove.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/item/configure/addto.phtml [hunk 16]
```diff
@@ -3,21 +3,25 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/item/configure/addto.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/item/configure/addto/wishlist.phtml [hunk 17]
```diff
@@ -3,12 +3,17 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/item/configure/addto/wishlist.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/item/list.phtml [hunk 18]
```diff
@@ -3,17 +3,20 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
-?>
-<?php
-/** @var \Magento\Wishlist\Block\Customer\Wishlist\Items $block */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/item/list.phtml:6]: missing new line above copyright****

File: app/code/Magento/Wishlist/view/frontend/templates/link.phtml [hunk 19]
```diff
@@ -3,11 +3,16 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/link.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/messages/addProductSuccessMessage.phtml [hunk 20]
```diff
@@ -3,8 +3,13 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/messages/addProductSuccessMessage.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/messages/removeWishlistItemSuccessMessage.phtml [hunk 21]
```diff
@@ -3,9 +3,14 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/messages/removeWishlistItemSuccessMessage.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/options_list.phtml [hunk 22]
```diff
@@ -3,19 +3,23 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/options_list.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/rss/email.phtml [hunk 23]
```diff
@@ -3,18 +3,24 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/rss/email.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/rss/wishlist.phtml [hunk 24]
```diff
@@ -3,11 +3,16 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/rss/wishlist.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/shared.phtml [hunk 25]
```diff
@@ -3,22 +3,27 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/shared.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/sharing.phtml [hunk 26]
```diff
@@ -3,36 +3,41 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/sharing.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/sidebar.phtml [hunk 27]
```diff
@@ -3,22 +3,25 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/sidebar.phtml:6]: missing new line above copyright

File: app/code/Magento/Wishlist/view/frontend/templates/view.phtml [hunk 28]
```diff
@@ -3,11 +3,16 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
```
Comment (leonhelmus) [inline app/code/Magento/Wishlist/view/frontend/templates/view.phtml:6]: missing new line above copyright

**Review by leonhelmus — APPROVED** [review]
Approved, but there are somethings which could have a newline. Nothing blocking in my opinion

**Pull request closed without merging**